cmd/broker/modules.go

```diff
@@ -132,9 +132,18 @@ func initModules(azureConfig config.AzureConfig) error {
 		azureEnvironment.ResourceManagerEndpoint,
 		azureSubscriptionID,
 	)
+
```

Comment:
Total nit: Let's remove this blank line and the one a few lines down from here. Visually, it's probably beneficial to keep related clients (e.g. all Postgres clients) grouped together.

Or ignore this suggestion. I realize how anal I'm being.